counting_sort/Java/count_sort.java

```diff
@@ -0,0 +1,64 @@
+import java.util.Arrays;
+import java.util.Collections;
+
+class CountSort {
+    /**
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpn7_d42aj/counting_sort/Java/count_sort.java
+++ b/tmp/tmpn7_d42aj/counting_sort/Java/count_sort.java
@@ -25,7 +25,7 @@
          */
         int maximum_number = getMax(arr);              // get the max number in the array in order to declare count array.
         int[] count_arr = new int[maximum_number + 1]; // maximum_number + 1 as we need to correctly
-                                                       //map each index with the number in the array 
+                                                       //map each index with the number in the array
 
         for (int i = 0; i < count_arr.length; i++) {
             count_arr[i] = 0;
```

Reply:
@sangamcse Hey Sangam, I am unable to apply patches. 
These are the errors I get ->
error: counting_sort/Java/CountSort.java: No such file or directory
error: patch failed: counting_sort/Java/count_sort.java:37
error: counting_sort/Java/count_sort.java: patch does not apply
error: counting_sort/Java/CountSort.java: No such file or directory
Can you please help me with this?